src/hooks/sdk/docs/useMediaStreamMetrics.stories.mdx

```diff
@@ -69,18 +86,30 @@ const App = () => (
 
 const MyChild = () => {
   const metrics = useMediaStreamMetrics();
+  const rtcStatsReportValues: ReactElement[] = [];
+
+  metrics.rtcStatsReport?.forEach((report) =>
+    rtcStatsReportValues.push(<pre>{JSON.stringify(report, null, 2)}</pre>)
+  );
 
   return (
-    <>
-      <p>Audio Uplink 1s loss: {metrics.audioPacketsSentFractionLossPercent}</p>
-      <p>
-        Audio Downlink 1s loss:{' '}
-        {metrics.audioPacketsReceivedFractionLossPercent}
-      </p>
-      <p>Bandwidth Incoming: {metrics.availableIncomingBandwidth}</p>
-      <p>Bandwidth Outgoing: {metrics.availableOutgoingBandwidth}</p>
-      <p>Video Metrics: {metrics.videoStreamMetrics}</p>
-    </>
+    <div>
+      <p>audioPacketsSentFractionLossPercent: {metrics.audioPacketsSentFractionLossPercent}</p>
+      <p>audioPacketsReceivedFractionLossPercent: {metrics.audioPacketsReceivedFractionLossPercent}</p>
+      <p>audioSpeakerDelayMs: {metrics.audioSpeakerDelayMs}</p>
+      <p>audioUpstreamRoundTripTimeMs: {metrics.audioUpstreamRoundTripTimeMs}</p>
+      <p>audioUpstreamJitterMs: {metrics.audioUpstreamJitterMs}</p>
+      <p>audioDownstreamJitterMs: {metrics.audioDownstreamJitterMs}</p>
+      <p>currentRoundTripTimeMs: {metrics.currentRoundTripTimeMs}</p>
+      <p>availableOutgoingBandwidth: {metrics.availableOutgoingBandwidth}</p>
+      <p>availableIncomingBandwidth: {metrics.availableIncomingBandwidth}</p>
+      <p>videoStreamMetrics</p>
```

Comment:
nit: videoStreamMetrics -> Video Stream Metrics?

Reply:
Well, I used the "Video Stream Metrics" at the first time, but the style is not consistent comparing to `audioPacketsSentFractionLossPercent`, ....
If we change videoStreamMetrics -> Video Stream Metrics, we should also change audioPacketsSentFractionLossPercent -> Audio Packets Sent Fraction Loss Percent. I don't really think it benefits us very much.

So I just use the property name of `useMediaStreamMetrics` hook here.